docs/content/documentation/modules/tensor_mechanics/C0TimoshenkoBeam.md

```diff
@@ -0,0 +1,192 @@
+
+## Dynamic beam
+There are two main ways to include the inertial effects for the beam. The first method is to assign a uniform density to the beam and calculate a consistent mass/inertia matrix for the beam, and the second method assumes the beam to be massless and adds point mass/inertia to the nodes at the end of the beam.
+
+### Consistent mass matrix
+If $A_y$, $A_z$ and $J$ are zero in [InertialForceBeam](/InertialForceBeam.md), then there is no coupling between the different dofs. The residual for the $j^{th}$ translational degree of freedom at node $i$ can be obtained as follows:
+$$
+R_j^0 = \int_{0}^{{}^0L} \rho A \left(\frac{x}{{}^0L} {\ddot{u}_j}^0 + \frac{{}^0L - x}{{}^0L} {\ddot{u}_j}^1 \right) \frac{x}{{}^0L} dx
+$$
+$$
+R_j^1 = \int_{0}^{{}^0L} \rho A \left(\frac{x}{{}^0L} {\ddot{u}_j}^0 + \frac{{}^0L - x}{{}^0L} {\ddot{u}_j}^1 \right) \frac{{}^0L - x}{{}^0L} dx
+$$
+where, $\rho$ is the density of the beam, which is assumed to be constant through the length of the beam, and ${\ddot{u}_j}^i$ is the translational acceleration at node $i$ in $j^{th}$ direction.  
+
+The residual for the $j^{th}$ rotational degree of freedom can be obtained as follows:
+$$
+R_j^0 = \int_{0}^{{}^0L} \rho I \left(\frac{x}{{}^0L} {\ddot{\theta}_j}^0 + \frac{{}^0L - x}{{}^0L} {\ddot{\theta}_j}^1 \right) \frac{x}{{}^0L} dx
+$$
+$$
+R_j^1 = \int_{0}^{{}^0L} \rho I \left(\frac{x}{{}^0L} {\ddot{\theta}_j}^0 + \frac{{}^0L - x}{{}^0L} {\ddot{\theta}_j}^1 \right) \frac{{}^0L - x}{{}^0L} dx
+$$
+where, $I = I_y + I_z$ for $j = 1$, $I = I_z$ for $j = 2$ and $I = I_y$ for $j = 3$.
+
+If $A_y$ and $A_z$ are non-zero, then there is coupling between $u_1$, $\theta_2$ and $\theta_3$, $u_2$ and $\theta_1$, and $u_3$ and $\theta_1$.
+
+### Point mass/inertia
+[NodalInertialForce](/NodalInertialForce.md) and [NodalInertialTorque](/NodalInertialTorque.md) are used to apply mass/inertia at a node. The mass and inertia terms contribute only to the residual of the node at which it is applied. Mass (m) behaves isotropically in all three coordinate directions resulting in the following $j^{th}$ translational nodal residual:
+$$
+R_j = m \ddot{u}_j
+$$
+
+[NodalInertialTorque](/NodalInertialTorque.md) takes 6 inertia components as input ($I_{xx}$, $I_{yy}$, $I_{zz}$, $I_{xy}$, $I_{xz}$, $I_{yz}$). If coordinate system in which the moment of inertia components are provided is different from the global coordinate system, then `x_orientation` and `y_orientation` need to be provided as input so that a transformation matrix from the local coordinate system to the global coordinate system can be calculated using which the moment of inertia tensor in the global coordinate system can then be obtained.
```

Comment:
Add 'the' after 'If'.

Also, the last part of the last sentence doesn't really make sense (the 'which' seems extraneous).